src/lib/serviceRoutinesV2/deleteEntity.cpp

```diff
@@ -91,28 +95,33 @@ std::string deleteEntity
       OrionError oe;
 
       ciP->httpStatusCode = scc;
+
       if (scc == SccContextElementNotFound)
       {
-        oe.code = scc;
-        oe.reasonPhrase ="NotFound";
-        oe.details = "The requested entity has not been found. Check type and id";
+        oe.code          = scc;
+        oe.reasonPhrase  = "NotFound";
+        oe.details       = "The requested entity has not been found. Check type and id";
       }
       else if (scc == SccInvalidParameter)
       {
-        oe.code = SccContextElementNotFound;
-        oe.reasonPhrase ="NotFound";
-        oe.details = "Attribute not found";
-        ciP->httpStatusCode = SccContextElementNotFound; // We don't want a 472
+        oe.code              = SccContextElementNotFound;
+        oe.reasonPhrase      = "NotFound";
+        oe.details           = "Attribute not found";
+        ciP->httpStatusCode  = SccContextElementNotFound; // We don't want a 472
       }
       else
       {
-        oe.code = scc;
-        oe.reasonPhrase = sc.reasonPhrase;
+        oe.code          = scc;
+        oe.reasonPhrase  = sc.reasonPhrase;
```

Comment:
Those lines are just indentations.
Apart from:

    #include "common/statistics.h"
    #include "common/clockFunctions.h"

But that is pretty clear ...
If we ever get a "style guide checker" in the makefile and fix all our code to follow the style-guide, these style-guide changes will no longer be necessary. Until then, they will continue. I know it bothers, but try to see it as improvements.

Not doing these improvements (trying to maintain a list of all of them until after LGTM) ... it's simply not practical.
Like that these improvements will never be done.

NTC